app/models/frontend/nics/ExemptionReason.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.frontend.nics
+
+import models.common.{Enumerable, WithName}
+
+sealed trait ExemptionReason
+
+object ExemptionReason extends Enumerable.Implicits {
```

Comment:
it's better to use `enumeratum` because it is standard in Scala 2 Community everywhere. Enumerable is just our custom implementation before we added Enumeratum to the project.